huaweicloud/utils/formatByConfig.go

```diff
@@ -0,0 +1,17 @@
+package utils
+
+import "regexp"
+
+const REPLACE_REG = "(?i)huaweicloud"
+
+var buildCloudCompany string
```

Comment:
just name this to cloud seems more clear, it's not specific to a company.

Reply:
suggest to rename the variable to **PackageName**, then we can define FLAGS in the makefile

```bash
PKG_NAME=huaweicloud
FLAGS=-ldflags="-X 'github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils.PackageName=$(PKG_NAME)'"
```

Reply:
I'm OK with both.

Reply:
The variable has bean renamed to PackageName